docs/source/model_design/risk_factors.rst

```diff
@@ -240,6 +240,88 @@ complexity into the exposure component.
 
 What is a risk effect?
 ++++++++++++++++++++++
+In epidemiological studies, risk effect is used to study the relationship
+between risk exposures and outcomes. We are not only interested in whether
+a link between a given risk exposure (e.g. smoking) and certain outcome
+(e.g. lung cancer) is statistically meaningful, but also the magnitude of
+this relationship. The effect of exposure can be measured both in relative
+and absolute terms. The risk ratio, the rate ratio, and the odds ratio are
+relative measures of effect. Risk difference is an absolute measure of effect
+and it is calculated by subtracting the risk of the outcome in exposed group
+from unexposed. [Measure_of_effect]_
+
+Risk effect in GBD
+^^^^^^^^^^^^^^^^^^
+The measure of risk effect in GBD is usually reported in relative term, namely
+relative risk. It describes the relative relationship between the risk of
+disease Y in the presence of agent X versus in absence of X. Mathematically,
+it's calculated by dividing the incidence rate (or other measure such as the
+excess mortality rate) of the cause in exposed population by the incidence rate
+(or other measure) of the cause in unexposed population for a certain risk factor.
+For example, if there are A incident cases and B person-years in exposed group;
+C incident cases and D person-years in unexposed group, then the relative risk
+(rate ratio) equals :math:`\frac{AD}{BC}`. Note that there are exceptions as in
+the low birth weight short gestation (LBWSG) risk factor where the relative risk
+is the ratio of all-cause mortality rate (ACMR) rather than incidence rate we
+mentioned above. Therefore, we'd better check with GBD modeller what relative
+risk they refer to before we model any risk-outcome pair in vivarium.
+
+Risk effect in vivarium
+^^^^^^^^^^^^^^^^^^^^^^^
+In vivarium, we used to build the risk-outcomes component in order to study the
+impact of desired outcomes contributed by given risk exposure. The outcome might
+be a cause (e.g. ischemic heart disease attributable to high body-mass index)
+or a intermediate outcome (e.g. systolic blood pressure associated with BMI).
+For a risk-cause pair, simulation model would link the incidence (or other measure
+such as excess mortality rate) of that cause to the relative risk from GBD or
+external data sources like literature evidence.
+
+The mathematical expressions are mainly fall into two categories:
+ - risk exposure is categorical distributed:
+     - :math:`i_{exposed} = i \times (1-PAF) \times RR`
+     - :math:`i_{unexposed} = i \times (1-PAF)`
+     - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
+     - :math:`E(RR_e) = p \times RR + (1-p)`
+ - risk exposure is continuous distributed:
+     - :math:`i = i \times (1-PAF) \times RR^{max(e−tmrel,0)/scalar}`
+     - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
+     - :math:`E(RR_e) = \int_{lb}^{ub}RR^{max(e−tmrel,0)/scalar}p(e)de`
```

Comment:
I'm still wary of using "RR" for the base of the exponential here rather than the result of the exponentiation, because I don't think it really makes sense to interpret the base as a relative risk, but perhaps we can change it later after getting some further clarification from @aflaxman.